tools/dds/poc/h/poc-h.cpp

```diff
@@ -72,10 +72,14 @@ int main( int argc, char** argv ) try
     TCLAP::CmdLine cmd( "POC host computer", ' ' );
     TCLAP::SwitchArg debug_arg( "", "debug", "Enable debug logging", false );
     TCLAP::ValueArg< realdds::dds_domain_id > domain_arg( "d", "domain", "select domain ID to listen on", false, 0, "0-232" );
+    TCLAP::SwitchArg op_pub_sub_arg( "o", "op-pub-sub", "create op pub and sub", false );
```

Comment:
Isn't it automatically false if the time-sync is 0?

Reply:
right. will not read/use time sync if no pub/sub